whylog_vim/gui/windows.py

```diff
@@ -69,7 +69,13 @@ def __init__(self):
         self.windows = dict()
 
     def create_window(self, window_type, content, modifiable=False, size=None):
+        self.origin_filename = VimUIWrapper.get_current_filename()
+        self.origin_offset = VimUIWrapper.get_cursor_offset()
         self.windows[window_type] = Window(window_type, content, modifiable, size)
+        for window in six.iterkeys(self.windows):
+            if self.origin_filename.endswith(window):
```

Comment:
at least we want to delete all windows with `/whylog_query`, so that `/home/user/foobar_whylog_query` is not matched. I also suspect that we can get an accurate match by concatenating result of vim's command `echo getcwd()` with a `/` and the file name.